docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -0,0 +1,405 @@
+
+<dt class="jump-anchor" id="gl-jwt">JSON Web Token (JWT)</dt>
+<dd>A JSON Web Token (JWT) is a type of bearer token, meaning that it contains all the information needed to allow the bearer to get access to a protected resource. With a JWT, the information is structured as a JSON object in a predefined format. It is URL-safe and digitally signed.</dd>
+
+<dt class="jump-anchor" id="gl-oidc-token">OpenID Connect (OIDC) token</dt>
+<dd>OpenID Connect (OIDC) is an identity layer on top of the OAuth 2.0 protocol that allows the client to verify the identity of an end-user based on authentication by an authorization server.</dd>
+<dd>In the context of Confidential Space, the Operator Service is the client and Confidential Space is the authorization server.</dd>
+
+<dt class="jump-anchor" id="gl-tee">TEE</dt>
+<dd>Acronym for Trusted Execution Environment. From the Confidential Space documentation:</dd>
+<dd>"Confidential Space uses a trusted execution environment that is designed to release secrets only to authorized workloads."</dd>
+<dd>For details, see <a href="https://cloud.google.com/docs/security/confidential-space#components-of-a-confidential-space-system">Components of a Confidential Space system</a> (Google Cloud documentation).</dd>
+
+<dt class="jump-anchor" id="gl-workload">Workload</dt>
+<dd>A containerized Docker image that runs in a cloud-based Trusted Execution Environment (see <a href="#gl-tee">TEE</a>) on top of the Confidential Space image.</dd>
+
+</dl>
+
+## Prerequisites
+
+Before setting up your UID2 Operator service in the Google Cloud Platform using Confidential Space, complete these prerequisite steps:
+
+- [Confidential Space Account Setup](#confidential-space-account-setup)
+- [UID2 Operator Account Setup](#uid2-operator-account-setup)
+
+### Confidential Space Account Setup
+
+You can run the UID2 Operator service on any GCP account and project. However, to support attestation, you'll need to create a service account that you can use to run Confidential Space virtual machines (VMs), and then grant specific permissions to the account.
+
+For the account setup, you'll need to provide valid values for the placeholders shown in the following tables.
+
+| Placeholder | Actual Value |
+| :--- | :--- |
+| `{PROJECT_ID}` | The ID of the GCP project that you want the GCP Operator to run in. |
+| `{SERVICE_ACCOUNT_NAME}` | A name you choose for this account. It's best to choose a clear but concise name. (**GWH_YS01 are there any naming constraints? I made that up re naming advice... we can take it out... just a guess.**)  |
+
+(**GWH_YS02 still not sure re PROJECT_ID... is this likely to be a GCP project that already exists, since the audience is someone who wants to set this up in GCP therefore already has an account? Or should setting up a GCP project be a step? (not that we tell them how to do it, but that we say do it). Not sure what the scope of a "GCP project" is.**)
```

Comment:
They could reuse the project id if they already have.
It depends on how customer organize. It's a way to manage resources.
My previous experience: we will create different project for different purpose.
We could recommend to create a seperated project to host uid2 private operator.

<img width="625" alt="image" src="https://github.com/IABTechLab/uid2docs/assets/137876002/4f2cb434-6c25-4d98-8759-c4049bc7b742">

Reply:
Updated. Thx!